tests/Makefile

```diff
@@ -112,9 +112,12 @@ BINARIES := $(addsuffix $(EXEXT),$(APPS))
 
 .SILENT:
 
-.PHONY: all check test clean
+.PHONY: all check test clean fuzz
 
-all: $(BINARIES)
+all: $(BINARIES) fuzz
+
```

Comment:
Please extend the `clean` target to recurse into `fuzz`. You can use this commit: https://github.com/gilles-peskine-arm/mbedtls/commit/3b2a1548632f32fded2a7d0cf6defbe0341e8ae2